docs/source/guide/Docker_README.md

```diff
@@ -87,11 +87,16 @@ These settings are shared with the repository workers
 
 Important: It should use the same db id as used by RSTUF Workers.
 
-#### (Required) `SECRETS_RSTUF_TOKEN_KEY`
+#### (Required) `RSTUF_AUTH`
+Disable/Enable RSTUF built-in token authentication. Default: true
+
+Note: Disable this feature only if you have set up another authentication provider.
```

Comment:
IMO we don't need this disclaimer/note. It will depend a lot on the architecture deployment. Maybe the API will be an internal service only, not exposed for use (might be a case for PyPI, for example)